content/pt/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Controladores
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Em robótica e automação um _control loop_, ou em português _ciclo de controle_, é
+um ciclo não terminado que regula o estado de um sistema.
+
+Um exemplo de ciclo de controlo é um termostato de uma sala.
+
+Quando você define a temperatura, isso indica ao termostato
+sobre o seu *estado desejado*. A temperatura ambiente real é o
+*estado atual*. O termostato atua de forma a trazer o estado atual
+mais perto do estado desejado, ligando ou desligando o equipamento.
+
+{{< glossary_definition term_id="controller" length="short">}}
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Padrão de controlo (Controller pattern)
```

Comment:
I'm not sure that *Controller pattern* translates to *Padrão de controlo*. I think it should something related to Controlador. Does it make sense?

Reply:
Totally, my mistake.